fast_and_reliable.py

```diff
@@ -0,0 +1,27 @@
+import time
+
+"""intit the list and the set"""
+words_list = []
+words_set = set()
+file = open('words.txt')
+for word in file:
+    words_list.append(word)
+    words_set.add(word)
+
+
+def search(words):
+    """ this function search the word "zwitterion"
+     in the given data structure
+     and return the time that took the function to preform the task  """
+    start = time.time()
```

Comment:
The start and end of what? it should be more informative.
Change those names please.